[2022-11-21 23:00:28,384] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: etl_example.transform_data scheduled__2022-11-21T22:00:00+00:00 [queued]>
[2022-11-21 23:00:28,412] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: etl_example.transform_data scheduled__2022-11-21T22:00:00+00:00 [queued]>
[2022-11-21 23:00:28,413] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-11-21 23:00:28,414] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2022-11-21 23:00:28,415] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-11-21 23:00:28,433] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): transform_data> on 2022-11-21 22:00:00+00:00
[2022-11-21 23:00:28,441] {standard_task_runner.py:52} INFO - Started process 3503 to run task
[2022-11-21 23:00:28,446] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'etl_example', 'transform_data', 'scheduled__2022-11-21T22:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/etl_example.py', '--cfg-path', '/tmp/tmpd_58kg0b', '--error-file', '/tmp/tmpy1j76ra7']
[2022-11-21 23:00:28,448] {standard_task_runner.py:77} INFO - Job 53: Subtask transform_data
[2022-11-21 23:00:28,517] {logging_mixin.py:109} INFO - Running <TaskInstance: etl_example.transform_data scheduled__2022-11-21T22:00:00+00:00 [running]> on host e6f0f1db9976
[2022-11-21 23:00:28,597] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_example
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-21T22:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-21T22:00:00+00:00
[2022-11-21 23:00:28,673] {python.py:152} INFO - Done. Returned value was: None
[2022-11-21 23:00:28,690] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=etl_example, task_id=transform_data, execution_date=20221121T220000, start_date=20221121T230028, end_date=20221121T230028
[2022-11-21 23:00:28,740] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-21 23:00:28,795] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
